2020-04-15 14:28:58 ERROR exception caught: org.springframework.http.converter.HttpMessageNotReadableException: Invalid JSON input: Cannot deserialize value of type `java.lang.Boolean` from String "8768": only "true" or "false" recognized; nested exception is com.fasterxml.jackson.databind.exc.InvalidFormatException: Cannot deserialize value of type `java.lang.Boolean` from String "8768": only "true" or "false" recognized
 at [Source: (PushbackInputStream); line: 3, column: 23] (through reference chain: hello.model.Customer["booking_confirmed"])
2020-04-15 14:31:09 ERROR exception caught: org.springframework.http.converter.HttpMessageNotReadableException: Invalid JSON input: Cannot deserialize value of type `java.lang.Boolean` from String "8768": only "true" or "false" recognized; nested exception is com.fasterxml.jackson.databind.exc.InvalidFormatException: Cannot deserialize value of type `java.lang.Boolean` from String "8768": only "true" or "false" recognized
 at [Source: (PushbackInputStream); line: 3, column: 23] (through reference chain: hello.model.Customer["booking_confirmed"])
2020-04-15 14:31:24 INFO  successful
2020-04-15 14:31:26 ERROR exception caught: hello.exceptions.NotFoundException: id 5e96f47ae1c7a73968e2e0e  not found
2020-04-15 14:31:37 ERROR exception caught: hello.exceptions.NotFoundException: Cannot find this ID
2020-04-15 14:31:56 INFO  successful
2020-04-15 14:32:26 INFO  successful
2020-04-15 14:32:42 ERROR exception caught: org.springframework.http.converter.HttpMessageNotReadableException: Invalid JSON input: Cannot deserialize value of type `java.lang.Boolean` from String "3": only "true" or "false" recognized; nested exception is com.fasterxml.jackson.databind.exc.InvalidFormatException: Cannot deserialize value of type `java.lang.Boolean` from String "3": only "true" or "false" recognized
 at [Source: (PushbackInputStream); line: 3, column: 23] (through reference chain: hello.model.Customer["booking_confirmed"])
2020-04-15 14:32:57 ERROR exception caught: hello.exceptions.NotFoundException: id 2  not found
2020-04-15 14:42:32 ERROR exception caught: hello.exceptions.NotFoundException: id 2  not found
2020-04-15 14:42:47 ERROR exception caught: hello.exceptions.NotFoundException: id 2  not found
2020-04-16 07:20:16 ERROR exception caught: org.springframework.dao.DataAccessResourceFailureException: Timed out after 30000 ms while waiting to connect. Client view of cluster state is {type=UNKNOWN, servers=[{address=chs-mongo:27017, type=UNKNOWN, state=CONNECTING, exception={com.mongodb.MongoSocketOpenException: Exception opening socket}, caused by {java.net.SocketTimeoutException: connect timed out}}]; nested exception is com.mongodb.MongoTimeoutException: Timed out after 30000 ms while waiting to connect. Client view of cluster state is {type=UNKNOWN, servers=[{address=chs-mongo:27017, type=UNKNOWN, state=CONNECTING, exception={com.mongodb.MongoSocketOpenException: Exception opening socket}, caused by {java.net.SocketTimeoutException: connect timed out}}]
2020-04-16 07:21:26 ERROR exception caught: org.springframework.dao.DataAccessResourceFailureException: Timed out after 30000 ms while waiting to connect. Client view of cluster state is {type=UNKNOWN, servers=[{address=chs-mongo:27017, type=UNKNOWN, state=CONNECTING, exception={com.mongodb.MongoSocketOpenException: Exception opening socket}, caused by {java.net.SocketTimeoutException: connect timed out}}]; nested exception is com.mongodb.MongoTimeoutException: Timed out after 30000 ms while waiting to connect. Client view of cluster state is {type=UNKNOWN, servers=[{address=chs-mongo:27017, type=UNKNOWN, state=CONNECTING, exception={com.mongodb.MongoSocketOpenException: Exception opening socket}, caused by {java.net.SocketTimeoutException: connect timed out}}]
2020-04-16 07:23:19 INFO  id5e956ef33887fb101b80e21efound
2020-04-16 07:24:28 INFO  id updated
